docs/examples/qaoa_qiskit.ipynb

```diff
@@ -0,0 +1,713 @@
+       "«     ┌────────────────┐┌─┴─┐ ░ └───┘└────────────────┘└───┘ ░ ┌─┴─┐»\n",
+       "«q_3: ┤ RZ(2*$\\gamma$) ├┤ X ├─░──────────────────────────────░─┤ X ├»\n",
+       "«     └────────────────┘└───┘ ░                              ░ └───┘»\n",
+       "«                             ░ \n",
+       "«q_0: ────────────────────────░─\n",
+       "«                             ░ \n",
+       "«q_1: ────────────────────────░─\n",
+       "«                             ░ \n",
+       "«q_2: ────────────────────■───░─\n",
+       "«     ┌────────────────┐┌─┴─┐ ░ \n",
+       "«q_3: ┤ RZ(2*$\\gamma$) ├┤ X ├─░─\n",
+       "«     └────────────────┘└───┘ ░ "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc_qaoa = QuantumCircuit(nqubits)\n",
+    "\n",
+    "qc_qaoa.append(qc_0, [i for i in range(0, nqubits)])\n",
+    "qc_qaoa.append(qc_mix, [i for i in range(0, nqubits)])\n",
+    "qc_qaoa.append(qc_p, [i for i in range(0, nqubits)])\n",
+    "\n",
+    "qc_qaoa.decompose().decompose().draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next step is to find the optimal parameters $(\\boldsymbol{\\beta_{opt}}, \\boldsymbol{\\gamma_{opt}})$ such that the expectation value\n",
+    "\n",
+    "$$ \\langle \\psi(\\boldsymbol{\\beta}_{opt}, \\boldsymbol{\\gamma}_{opt}) \\rvert H_P \\lvert \n",
+    "\\psi(\\boldsymbol{\\beta}_{opt}, \\boldsymbol{\\gamma}_{opt}) \\rangle $$\n",
+    "\n",
+    "is minimized. Such an expectation can be obtained by doing measurement in the Z-basis. We use a classical optimization algorithm to find the optimal parameters. Following steps are involved as shown in the schematic \n",
+    "<img src=\"https://qiskit.org/textbook/ch-applications/images/QAOA_schematic.png\" width=\"400\" height=\"200\" class=\"center\">\n",
+    "\n",
+    "1. Initialize $\\boldsymbol{\\beta}$ and $\\boldsymbol{\\gamma}$ to suitable real values.\n",
+    "2. Repeat until some suitable convergence criteria is met:\n",
+    "    1. Prepare the state $\\lvert \\psi(\\boldsymbol{\\beta}, \\boldsymbol{\\gamma}) \\rangle$ using qaoa circuit\n",
+    "    2. Measure the state in standard basis\n",
+    "    3. Compute $\\langle \\psi(\\boldsymbol{\\beta}, \\boldsymbol{\\gamma}) \\rvert H_P \\lvert \\psi(\\boldsymbol{\\beta}, \\boldsymbol{\\gamma}) \\rangle$ \n",
+    "    4. Find new set of parameters $(\\boldsymbol{\\beta}_{new}, \\boldsymbol{\\gamma}_{new})$ using a classical optimization algorithm\n",
+    "    5. Set current parameters $(\\boldsymbol{\\beta}, \\boldsymbol{\\gamma})$ equal to the new parameters \n",
+    "      $(\\boldsymbol{\\beta}_{new}, \\boldsymbol{\\gamma}_{new})$\n",
+    "\n",
+    "The code below implements the steps mentioned above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
We also don't seem to be getting any Python syntax highlighting here... but I'm guessing that's the ipython lexer error?

Reply:
Fixed!